packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -635,7 +582,7 @@ async function startCompass(
   let browser: CompassBrowser;
 
   try {
-    browser = await remote(options);
+    browser = (await remote(options)) as CompassBrowser;
```

Comment:
I'm casting here because we add those extra custom browser commands and they are there. Not 100% sure why it wasn't necessary before, though.